storage-logzio/src/main/java/zipkin2/storage/logzio/LogzioStorage.java

```diff
@@ -69,18 +68,21 @@ public boolean isStrictTraceId() {
         return strictTraceId;
     }
 
+    volatile HttpCall.Factory http;
+
     // hosts resolution might imply a network call, and we might make a new okhttp instance
-    @Memoized
```

Comment:
this change is a fix. while you added `@Memoized`, since the type wasn't auto-value, it actually did nothing (wouldn't have remembered the field). I implemented memoization manually here as it is less code than switching the whole type to auto-value.